outline/shadowsocks/connectivity.go

```diff
@@ -37,7 +36,8 @@ func CheckConnectivity(client *Client) (int, error) {
 	// Start asynchronous UDP support check.
 	udpChan := make(chan error)
 	go func() {
-		udpChan <- oss.CheckUDPConnectivityWithDNS(client, shadowsocks.NewAddr("1.1.1.1:53", "udp"))
+		resolverAddr := &net.UDPAddr{IP: net.ParseIP("1.1.1.1"), Port: 53}
```

Comment:
alternative [`ResolveUDPAddr`](https://pkg.go.dev/net#ResolveUDPAddr)

Reply:
Yeah, I thought about using that, but then I need to add a comment saying there's no resolution going on. I find this clearer.